source/ExpandablePicker.js

```diff
@@ -553,6 +553,25 @@
 			}
 			this.$.client.setHighlander(!this.multipleSelection);
 			this.selectedChanged();
+		},
+
+		/**
+		* If drawer is currently open, and event was sent via keypress (i.e., it has a direction),
+		* process header focus.
+		*
+		* @fires moon.Scroller#onRequestScrollIntoView
+		* @private
+		*/
+		headerFocus: function (inSender, inEvent) {
```

Comment:
Hi @aarontam , sorry for the late response. Your solution looks clean and simple.